tests/FakeItEasy.IntegrationTests/TypeCatalogueTests.cs

```diff
@@ -150,7 +150,7 @@ private static string CaptureConsoleOutput(Action action)
 
         private string GetPathToOriginalExternalDll()
         {
-            var executingAssemblyPath = new Uri(this.GetType().GetTypeInfo().Assembly.Location).LocalPath
+            var executingAssemblyPath = new Uri(this.GetType().GetTypeInfo().Assembly.CodeBase).LocalPath

```

Comment:
To be honest, I had missed the `.Location` on line 161 :wink:
But after looking at it, I think it should stay as is.